readthedocs/projects/models.py

```diff
@@ -1430,9 +1430,21 @@ class Meta:
     objects = HTMLFileManager()
 
     def get_processed_json(self):
-        parser_class = (
-            SphinxParser if self.version.is_sphinx_type else MkDocsParser
-        )
+        if (
+            self.version.documentation_type == constants.GENERIC
+            or self.project.has_feature(Feature.INDEX_FROM_HTML_FILES)
```

Comment:
Are we using this feature much? Was it user-requested or just for testing?

Reply:
It was for testing, mostly as an alternative for mkdocs (where we index using their json file, but indexing directly from the html gives slightly better results)

Reply:
I think it's still useful, specially as an alternative for mkdocs (and as a replacement in the future)